tests/testthat/test-derive_param_clinbenefit.R

```diff
@@ -27,15 +34,15 @@ adrs <- tribble(
   "04",     "RSP",    "N",    NA,
   "06",     "RSP",    "N",    NA,
   "01",     "PD",     "N",    NA,
-  "02",     "PD",     "Y",    "2021-05-07",
+  "02",     "PD",     "Y",    "2021-03-06", # changed to "2021-03-06", sgorm123
```

Comment:
we don't need the comments here as we have the git history and blame to rely on to be able to trace all updates. if we used this convention our tests would become unreadable over time given so many developers and updates over time

Reply:
I understand, however, I did this for the original author of the code, as I changed their test data (and was concerned). I can remove no problem. Though, are you ok with the header in the same test file:

![image](https://user-images.githubusercontent.com/31095773/188478316-bbf77694-74b0-4954-a9a3-3e6c0018c67c.png)

Reply:
I agree with @rossfarrugia . Comments should describe the current code but not the history. After the next update of the file it is no longer clear to which change the comment/header refers.

Reply:
yes i'd wipe that header bit describing the change too - version change info like this is better stored in git history via issues and PRs rather than within the code itself

Reply:
OK, I will remove, maybe I could ask @amsmith214 to double check too, as the original author. If you have time @amsmith214

Reply:
Andrew has left GSK now so won't be contributing anymore. Stefan's review will be sufficient